### PR TITLE
PR to sync with AiA updates from https://github.com/riscv/riscv-aia/pull/28

### PR DESCRIPTION
**Pull request by ved-rivos**

### CODE REVIEW

File: iommu_data_structures.adoc [hunk 1]
```diff
@@ -619,22 +620,22 @@ The `MODE` field is used to select the MSI address translation scheme.
 [%header, cols="3,3,20"]
 |===
 |Value | Name     | Description
-| 0    | `Bare`   | No translation or protection. MSI recognition using
-                    MSI address mask and pattern is not performed.
+| 0    | `Bare`   | No translation or protection. Recognition of accesses to
+                    a virtual interrup file using MSI address mask and
```
Comment (pperesse) [inline iommu_data_structures.adoc:624]: interrupt (t is missing)

Comment (pperesse) [reply iommu_data_structures.adoc:624]: rest looks good

Comment (ved-rivos) [reply iommu_data_structures.adoc]: thanks. fixed!

**Comment by ved-rivos** [comment]
Discussed in the IOMMU TG. Agreed that duplicating text like the page table formats and MRIF layout where the IOMMU spec does not add any further clarification or contents is not desirable. This PR replaces such sections with a reference the AIA specification.

**Pull request merged**